packages/trader/src/App/Components/Elements/ContractAudit/contract-audit.jsx

```diff
@@ -3,88 +3,151 @@ import React from 'react';
 import { Icon, ThemedScrollbars } from '@deriv/components';
 import { localize } from '@deriv/translations';
 import { epochToMoment, toGMTFormat } from 'Utils/Date';
-import { getBarrierLabel, getBarrierValue, isDigitType } from 'App/Components/Elements/PositionsDrawer/helpers';
+import { addCommaToNumber, getBarrierLabel, getBarrierValue } from 'App/Components/Elements/PositionsDrawer/helpers';
 import ContractAuditItem from './contract-audit-item.jsx';
 
 class ContractAudit extends React.PureComponent {
     render() {
         const { contract_end_time, contract_info, duration, duration_unit, exit_spot, has_result } = this.props;
+
         if (!has_result) return null;
+
+        // Contract type checks
+        const is_digit = /DIGIT/.test(contract_info.contract_type);
```

Comment:
please use a helper function `isDigitContract` from  Stores/Modules/Contract/Helpers/digits.js :)

Reply:
@kelly-binary for this please look at the other comment. `isDigitContract` is also removed for the same reason.